README.asciidoc

```diff
@@ -215,16 +215,20 @@ Install plugin from rubygems:
 $ vagrant plugin install vagrant-openstack-plugin
 ----
 
+NOTE: On some systems (e.g. mac) doing `export NOKOGIRI_USE_SYSTEM_LIBRARIES=1` can help getting the above command working.
```

Comment:
Minor change.  How about:

NOTE: On some systems (e.g. mac), `export NOKOGIRI_USE_SYSTEM_LIBRARIES=1` can help make the above command work.